### ISSUE TITLE
Check users with placeholder passwords

### ISSUE THREAD

**Issue by joecorcoran** [rubymonsters/diversity_ticketing#280]
Madeleine from RuhrJS couldn't log in today, and eventually had to do a password reset to resolve the problem.

I noticed that before the password was reset, the `encrypted_password` was set to a single character.

Was this a consequence of having to create user accounts by hand, after the authentication system was added? If so, do we need to reset other accounts?

cc @ariaru @fidothe

**Comment by ariaru** [comment]
Password reset is way to get access to the accounts that we assigned to event organizers that had used diversitytickets.org in the past that we intended. I had asked @anikalindtner to send out emails to them explaining this, but I guess she forgot or decided it wasn't necessary(?). So for every event organizer email address from the past, there is one user account, with an impossible password (no password one might choose would turn out this way when encrypted), that the person who owns the email address can reset to then be able to use the account.
